src/FileManager.php

```diff
@@ -29,14 +29,16 @@ class FileManager
     private $rootDirectory;
     /** @var SymfonyStyle */
     private $io;
+    private $twigDefaultPath;
 
-    public function __construct(Filesystem $fs, AutoloaderUtil $autoloaderUtil, string $rootDirectory)
+    public function __construct(Filesystem $fs, AutoloaderUtil $autoloaderUtil, string $rootDirectory, string $twigDefaultPath)
     {
         // move FileManagerTest stuff
         // update EntityRegeneratorTest to mock the autoloader
         $this->fs = $fs;
         $this->autoloaderUtil = $autoloaderUtil;
         $this->rootDirectory = rtrim($this->realPath($this->normalizeSlashes($rootDirectory)), '/');
+        $this->twigDefaultPath = $twigDefaultPath;
```

Comment:
There's on problem here: the `%twig.default_path%` that is passed to this method is an absolute URL to your system, not just the string `templates/`. That's a problem (at the very least) because we use this path in some messages - e.g. `Review & adapt the login template... `

I'm not 100% sure, but I think we could just convert this absolute path to the relative path (e.g. `templates`) and set the `twigDefaultPath` property to just that string. I think the rest of the system is perfectly happy just using this relative path, because the FileManager or Generator ultimately converts it to the absolute path later before generation. We could make that conversion using the `relativizePath()` method.

Reply:
Done @weaverryan !
(Sorry for the rebase...)